index.html

```diff
@@ -1518,44 +1518,23 @@ <h2 id="state-transitions" class="informative">
     </section>
     <section>
       <h2>
-        PaymentRequest and iframes
+        PaymentRequest and <code>iframe</code> elements
       </h2>
       <p>
-        There are some circumstances where a cross-origin <a>iframe</a> wants
-        to make a payment request. A cross-origin iframe needs explicit
-        permission from the embedding page to invoke the payment request API.
+        There are some circumstances where a cross-origin
+        <a><code>iframe</code></a> wants to make a payment request. A
+        cross-origin <a><code>iframe</code></a> needs explicit permission
+        from the embedding page to invoke the payment request API.
       </p>
       <p>
-        The <a>HTMLIFrameElement</a> is extended with an
-        <dfn><code>allowpaymentrequest</code></dfn> content attribute.
-        <a>allowpaymentrequest</a> is a <a>boolean attribute</a>. When
-        specified, it indicates that scripts in the iframe element's browsing
-        context are <dfn>allowed to make payment requests</dfn> (if it's not
-        blocked for other reasons, e.g., there is another ancestor iframe
-        without this attribute set).
+        When an <a><code>iframe</code></a> element has an
+        <a><code>allowpaymentrequest</code></a> attribute, and does not
+        have any ancestor <a><code>iframe</code></a> element without an
```

Comment:
This seems wrong. This reads like it's talking about ancestor the DOM concept. But you should hook into "allowed to use" instead, which handles nested browsing contexts. See Fullscreen.